src/webgpu/shader/validation/expression/call/builtin/atomics.spec.ts

```diff
@@ -0,0 +1,66 @@
+export const description = `
+Validation tests for atomic builtins.
+`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { ShaderValidationTest } from '../../../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const atomicOps = [
+  'atomicAdd(&a, 1)',
+  'atomicSub(&a, 1)',
+  'atomicMax(&a, 1)',
+  'atomicMin(&a, 1)',
+  'atomicAnd(&a, 1)',
+  'atomicOr(&a, 1)',
+  'atomicXor(&a, 1)',
+  'atomicLoad(&a)',
+  'atomicStore(&a, 1)',
```

Comment:
Can you add an `unimplemented` test to check `atomicExchange`, and `atomicCompareExchangeWeak` as well?

Reply:
Thanks for pointing this out, I actually had forgotten to add these two to the list.